pages/02.Setup/01.Getting Started/docs.en.md

```diff
@@ -109,23 +109,23 @@ The next step after creating the database is to set up your administrator accoun
 
 ![admin user](admin-user.png)
 
-Enter a username, password, first and last name and an email address, and the account will be created for you.
+Enter a username, password, first and last name, and an email address, and the account will be created for you.
 
 ### Email configuration
 
 The next step allows you to configure the way in which email is sent from your Mautic instance.
 
 ![email config](email-config.png)
 
-If you are testing, whether locally or on a live server, it is highly recommended to make use of [Mailhog](https://github.com/mailhog/MailHog) or similar to capture all outgoing emails, until you are ready to send to real recipients. Otherwise, in a production environment you will need to choose whether you want to send email immediately, or queue email (and process it when the Cron job runs), and provide the appropriate server, port, encryption type, credentials and authentication mode for the systems you wish to use.
+If you are testing, whether locally or on a live server, it is highly recommended to make use of [Mailhog](https://github.com/mailhog/MailHog) or similar to capture all outgoing emails, until you are ready to send to real recipients. Otherwise, in a production environment you will need to choose whether you want to send an email immediately, or queue email (and process it when the Cron job runs), and provide the appropriate server, port, encryption type, credentials, and authentication mode for the systems you wish to use.
 
 ### Admin login
 
 Now you have completed the setup process, you will be directed to the back-end login screen at `http(s)://example.com/mautic/s/login` - this is where you log into your Mautic to manage your instance, so it's worth bookmarking the login page for future reference.
 
 ## Step 2: Add Cron Jobs
 
-Once you've installed Mautic you will need to create a few standard [cron jobs][cron] to have your software process various tasks. These [cron jobs][cron] can be created through a cPanel or added through command line. Here is a list of the [cron jobs][cron] you'll need to create. (Please note: "/path/to/mautic..." will reflect the directory where your version of Mautic is installed, and you may need to replace 'php' with the path to PHP on your server. Ask your hosting provider if you are not sure.)
+Once you've installed Mautic you will need to create a few standards [cron jobs][cron] to have your software process various tasks. These [cron jobs][cron] can be created through a cPanel or added through the command line. Here is a list of the [cron jobs][cron] you'll need to create. (Please note: "/path/to/mautic..." will reflect the directory where your version of Mautic is installed, and you may need to replace 'php' with the path to PHP on your server. Ask your hosting provider if you are not sure.)
```

Comment:
In the first case, we are referring to the standard [cron jobs listed in the docs](https://docs.mautic.org/en/setup/cron-jobs), so it should be singular, as we are not referring to a set of standards, but the basic cron jobs that Mautic needs to function.

Second fix is perfect, great catch! :eyes: